webpack.config.js

```diff
@@ -8,6 +8,8 @@ const htmlPlugin = new HtmlWebPackPlugin({
 });
 
 module.exports = {
+  devServer: { historyApiFallback: true },
```

Comment:
Why do we have this here

Reply:
it was suggested as a fix to `Content Security Policy: The page's settings blocked the loading of a resource at eval ("default-src")`

see https://stackoverflow.com/questions/46422290/webpack-dev-server-react-content-security-policy-error?answertab=active#tab-top

Reply:
ok